.github/workflows/test.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
         touch "$CACHE_RESULT_STAMP"
 
   unit-tests:
-    runs-on: ubuntu-16.04
+    runs-on: ubuntu-20.04
```

Comment:
this is fine too, since we will also run unit tests on xenial/trusty/others via spread anyways